docs/source/markdown/podman-attach.1.md

```diff
@@ -15,7 +15,7 @@ or name, either to view its ongoing output or to control it interactively.
 You can detach from the container (and leave it running) using a configurable key sequence. The default
 sequence is `ctrl-p,ctrl-q`.
 Configure the keys sequence using the **--detach-keys** option, or specifying
-it in the **libpod.conf** file: see **libpod.conf(5)** for more information.
+it in the **containers.conf** file: see **containers.conf(5)** for more information.
```

Comment:
Probably should add container.conf(5) to the bottom of this page.

Reply:
Done